src/multiversioning.zig

```diff
@@ -600,8 +664,7 @@ pub const Multiversion = struct {
     target_body_offset: ?u32 = null,
     target_body_size: ?u32 = null,
     target_header: ?MultiversionHeader = null,
-    /// This list is referenced by `Replica.releases_bundled`.
-    /// Note that this only contains the advertisable releases, which are a subset of the actual
+    /// This list  only contains the advertisable releases, which are a subset of the actual
```

Comment:
```suggestion
    /// This list only contains the advertisable releases, which are a subset of the actual
```

Übernit, sorry :rofl: